contrib/node/src/python/pants/contrib/node/tasks/node_test.py

```diff
@@ -65,14 +70,29 @@ def _execute(self, all_targets):
     for target in targets:
       node_path = node_paths.node_path(target.dependencies[0])
 
-      args = ['run-script', target.script_name, '--'] + self.get_passthru_args()
-
-      with pushd(node_path):
-        self._currently_executing_test_targets = [target]
-        result, npm_test_command = self.execute_npm(args, workunit_labels=[WorkUnitLabel.TEST])
-        if result != 0:
-          raise TaskError('npm test script failed:\n'
-                          '\t{} failed with exit code {}'.format(npm_test_command, result))
+      logger.debug('First dependency: %s', target.dependencies[0])
```

Comment:
This implementation looks super incompatible with pants because node package only has one package.json stay in the package folder. So we simply picked the first dependency as the package to be tested, the second will be dropped. If you have two node packages to be tested. You have to write two separate testing target for them.